README.md

```diff
@@ -281,7 +282,7 @@ The following table describes the environment variables. By default, the SDK "na
 | Variable name           | Description                                                                                                                                                           |
 | ----------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | LOOKERSDK_BASE_URL      | A URL like `https://my.looker.com:19999`. No default value.                                                                                                           |
-| LOOKERSDK_API_VERSION   | Version of the Looker API to use. Use `3.1` for now, which is the default and used to produce this SDK.                                                               |
+| LOOKERSDK_API_VERSION   | Version of the Looker API to use. Use `4.0` for the current API version.                                                                                              |
```

Comment:
I wonder where we're actually using this now. Worth spending a few minutes verifying whether we still use it or not.

Reply:
Actually used in only 1 place:
- In a "check'' against the API login endpoint during a "wait for looker" github script.

Programmatically passed-through / typed in 2 places
- Docker config passes it around
- Go type delarations include it

Doc'd in 4 places:
- This main readme
- Two env sample files
- A Python notebook example

From the one use, I think the default value (4.0) should be fine (and indistinguishable in functionality from 3.1) in virtually every situation besides trying to build against a _very_ old Looker version, so maybe we leave the variable there but remove it from the documentation?

Reply:
I think removing it from the doc is a good step